packages/shared/src/components/modals/award/BuyCoresModal.tsx

```diff
@@ -202,9 +248,19 @@ export const BuyCoresProcessing = ({ ...props }: ModalProps): ReactElement => {
       const maxRetries = (oneMinute * 1000) / transactionRefetchIntervalMs;
 
       if (retries > maxRetries) {
-        // TODO feat/transactions redirect user to /wallet to monitor their transaction there if they want
         // log error for timeout
         onAskConfirmation(false);
+        setActiveStep({
+          step: 'PROCESSING_ERROR',
+        });
+        setProcessingError({
+          title: 'Processing timed out',
+          description:
+            'Please check the status of your transaction in your Core Wallet.',
```

Comment:
Just check with product if "Core Wallet" is right capitalization

Reply:
it is yes 👍